lib/elixir/pages/anti-patterns/process-anti-patterns.md

```diff
@@ -76,7 +76,111 @@ iex> Calculator.subtract(2, 3)
 
 ## Scattered process interfaces
 
-TODO.
+#### Note
+
+Formerly known as [Agent obsession](https://github.com/lucasvegi/Elixir-Code-Smells/tree/main#agent-obsession).
+
+#### Problem
+
+In Elixir, the use of an `Agent`, a `GenServer`, or any other process abstractions is not an anti-pattern in itself. However, when the responsibility for direct interaction with a process is spread throughout the entire system, it can become problematic. This bad practice can increase the difficulty of code maintenance and make the code more prone to bugs.
```

Comment:
```suggestion
In Elixir, the use of an `Agent`, a `GenServer`, or any other process abstraction is not an anti-pattern in itself. However, when the responsibility for direct interaction with a process is spread throughout the entire system, it can become problematic. This bad practice can increase the difficulty of code maintenance and make the code more prone to bugs.
```